sdk/keyvault/keyvault-keys/src/lro/recover/operation.ts

```diff
@@ -0,0 +1,103 @@
+import { AbortSignalLike } from "@azure/abort-controller";
```

Comment:
Something I just noticed but larger scope than this PR: shouldn't all source have appropriate MIT license headers? We have to in .NET (we have code analysis rules now to make sure we do and that they have the CELA-approved format).